.github/workflows/canary.yml

```diff
@@ -11,6 +11,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@master
+
+      - name: Embed version
+        run: |
+          echo ${GITHUB_SHA:0:7} > lua/wire/version.lua
```

Comment:
Does this really exist? Where are you getting this from?

Reply:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

It worked when I test ran it.